stdlib/websub/src/main/ballerina/src/websub/publisher_client.bal

```diff
@@ -24,20 +24,26 @@ public type PublisherClient client object {
     private string url;
     private http:Client httpClient;
 
-    # Initializer function for the client.
-    #
-    # + url    - The URL to publish/notify updates to
-    # + config - The `http:ClientConfiguration` for the underlying client or `()`
+# Initializes the `websub:PublisherClient`.
+# ```ballerina
+# websub:PublisherClient websubHubClientEP = new ("http://localhost:9191/websub/publish");
+# ```
+#
+# + url    - The URL to publish/notify updates to
+# + config - The `http:ClientConfiguration` for the underlying client or else `()`
     public function __init(string url, http:ClientConfiguration? config = ()) {
         self.url = url;
         self.httpClient = new (self.url, config);
     }
 
-    # Registers a topic in a Ballerina WebSub Hub against which subscribers can subscribe and the publisher will
-    # publish updates.
-    #
-    # + topic - The topic to register
-    # + return - `error` if an error occurred registering the topic
+# Registers a topic in a Ballerina WebSub Hub against which subscribers can subscribe and the publisher will
+# publish updates.
+# ```ballerina
+# error? registerTopic = websubHubClientEP->registerTopic("http://websubpubtopic.com");
+# ```
+#
+# + topic - The topic to register
+# + return - An `error` if an error occurred registering the topic or esle `()`
```

Comment:
```suggestion
# + return - An `error` if an error occurred registering the topic or esle `()`
```
Need to specify the error.

Reply:
We are using a common error in this module. So we don't need to specify the error.